src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -47,29 +49,52 @@ const ArtworkPreviewContainer = styled.View`
   marginBottom: 10
 `
 
-interface Props {
-  message: {
-    senderName: string
-    time: string
-    body: string
-  }
+const ImagePreviewContainer = styled.View`
+  marginBottom: 10
+`
+
+interface Props extends RelayProps {
+  senderName: string
   artworkPreview?: JSX.Element
+  imagePreview?: JSX.Element
 }
 
-export default class Message extends React.Component<Props, any> {
+export class Message extends React.Component<Props, any> {
   render() {
+    const date = moment(this.props.message.created_at).fromNow(true)
     return (
       <Container>
         <Avatar />
         <TextContainer>
           <Header>
-            <SenderName>{this.props.message.senderName}</SenderName>
-            <MetadataText>{this.props.message.time}</MetadataText>
+            <SenderName>{this.props.senderName}</SenderName>
+            <MetadataText>{date}</MetadataText>
           </Header>
           {this.props.artworkPreview && <ArtworkPreviewContainer>{this.props.artworkPreview}</ArtworkPreviewContainer>}
-          <BodyText>{this.props.message.body.split("\n\nAbout")[0]}</BodyText>
+          {this.props.imagePreview && <ImagePreviewContainer>{this.props.imagePreview}</ImagePreviewContainer>}
+          <BodyText>{this.props.message.raw_text.split("\n\nAbout")[0]}</BodyText>
```

Comment:
Is `this.props.message.raw_text.split("\n\nAbout")[0]` worth verifying that the split will definitely work? Can't trust the server_too_ much ;)